ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/miner/MinerSetMinPriorityFee.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.api.jsonrpc.internal.methods.miner;
+
+import org.hyperledger.besu.datatypes.Wei;
+import org.hyperledger.besu.ethereum.api.jsonrpc.RpcMethod;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.JsonRpcRequestContext;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.methods.JsonRpcMethod;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.response.JsonRpcResponse;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.response.JsonRpcSuccessResponse;
+import org.hyperledger.besu.ethereum.core.MiningParameters;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MinerSetMinPriorityFee implements JsonRpcMethod {
+  private static final Logger LOG = LoggerFactory.getLogger(MinerSetMinPriorityFee.class);
+
+  private final MiningParameters miningParameters;
+
+  public MinerSetMinPriorityFee(final MiningParameters miningParameters) {
+    this.miningParameters = miningParameters;
+  }
+
+  @Override
+  public String getName() {
+    return RpcMethod.MINER_SET_MIN_PRIORITY_FEE.getMethodName();
+  }
+
+  @Override
+  public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
+    try {
+      final Wei minPriorityFeePerGas = Wei.of(requestContext.getRequiredParameter(0, Long.class));
```

Comment:
There's a slight concern on the signed Long usage for Wei, which is much smaller than `UInt256`, which is what under the hood of Wei can is a theoretical maximum.